mobile/src/libs/components/modal/modal.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Modal as RNModal } from 'react-native';
+
+import { Button, Pressable, Text, View } from '#libs/components/components';
+
+import { styles } from './styles';
+
+type Properties = {
+  isVisible: boolean;
+  onClose: () => void;
+  onDelete?: () => void;
```

Comment:
Maybe it's better to name it `onSubmit`? All in all it's common component, in local case submiting will be the deletion

Reply:
@Dmytro-Kucherenko this component is used for adding meditation entries, so i guess its better leave onDelete